Code/Engine/Foundation/Platform/Android/Screen_Android.cpp

```diff
@@ -1,11 +1,22 @@
 #include <Foundation/FoundationPCH.h>
 
-#if EZ_ENABLED(EZ_PLATFORM_ANDROID)
+#include <Foundation/Basics/Platform/PlatformFeatures.h>
+#include <Foundation/System/Screen.h>
 
-#  if EZ_ENABLED(EZ_SUPPORTS_GLFW)
-#    include <Foundation/Platform/GLFW/Screen_GLFW.h>
-#  else
-#    include <Foundation/Platform/NoImpl/Screen_NoImpl.h>
-#  endif
+#if EZ_ENABLED(EZ_PLATFORM_ANDROID)
 
+ezResult ezScreen::EnumerateScreens(ezHybridArray<ezScreenInfo, 2>& out_Screens)
+{
+  if (ANativeWindow* pWindow = ezAndroidUtils::GetAndroidApp()->window)
+  {
+    ezScreenInfo& currentScreen = out_Screens.ExpandAndGetRef();
+    currentScreen.m_sDisplayName = "Current Display";
+    currentScreen.m_iOffsetX = 0;
+    currentScreen.m_iOffsetY = 0;
+    currentScreen.m_iResolutionX = ANativeWindow_getWidth(pWindow);
+    currentScreen.m_iResolutionY = ANativeWindow_getHeight(pWindow);
+    currentScreen.m_bIsPrimary = true;
+  }
+  return EZ_SUCCESS;
```

Comment:
Wouldn't it be a failure, if there is no screen ?

Reply:
Good point, wasn't sure what the result means, the other implementations also fail on 0 monitors so will change it.